triton-client/.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = false
```

Comment:
`trim_trailing_whitespace` should be `true` if you plan to keep this config.

Reply:
np

Reply:
It's still false ;)

Reply:
Oh my bad you might not be done yet with your changes

Reply:
oh no, i just missed it, there are a lot of files and im jumping back and forth on subjects, good catch